spec/models/user_spec.rb

```diff
@@ -60,6 +60,14 @@
     it 'returns the correct name' do
       expect(user.full_name).to eq('John Doe')
     end
+
+    context 'when test_feature feature flag is on' do
+      before { Flipper.enable(:test_feature) }
+
+      it 'returns the last_name first' do
+        expect(user.full_name).to eq('Doe John')
+      end
+    end
```

Comment:
I'll remove this change before merging the PR, this is just a small POC within the PR